components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+
+// Common components
+import { SettingsText } from '../../../components/default'
+import { Toggle } from '../../../components/toggle'
+
+// Assets
+import InfoIcon from '../../../components/default/settings/assets/info-icon.svg'
+
+// Utilities
+import { getLocale } from '../../../../common/locale'
+
+const isDarkTheme = (p: any) => {
+  return p.theme.name === 'Brave Dark'
+}
+
+interface Props {
+  onChange: () => void
+  onEnableRewards: () => void
+
+  checked: boolean
+  disabled: boolean
+  rewardEnabled: boolean
+  adsEnabled: boolean
+  canSupportAds: boolean
+}
+
+const Container = styled('div')<{}>`
+  background-color: rgba(93, 181, 252, 0.2);
```

Comment:
I take it we aren't lucky enough to have a CSS variable for this color? @petemill is it worth adding the color to `app.global.scss`?

Reply:
a

Reply:
Yeah, so the new design system isn't in Brave yet (though hopefully soon)! For now, I think it's probably fine to either leave this as is, or add a variable to `app.global.scss`. Up to you :smile:

Reply:
Cool! Done :)